README.md

```diff
@@ -78,6 +78,9 @@ Clone infrastructure repository:
 
 NOTE: VM will boot from volume, you no longer have to attach additional volumes. Size of the volume can be set by `-e vm_size=100`
 
+    Install the various ansible roles
+    (dev) $ ansible-galaxy install -r requirements.yml
```

Comment:
true, I was going for a more generic one if people were going to use the infrastructure